web/content/js/nav.js

```diff
@@ -67,15 +58,57 @@ let navItems = [
 		href: "/Application_Centric_Deployment/index-foreman-el.html"
 	}]
 },{
-	title: "Katello documentation", items: [
+	title: "Foreman on Debian", items: [
+	{
+		title: "Release Notes",
+		href: "/Release_notes/index-foreman-deb.html"
+	},{
+		title: "Quickstart Guide",
+		href: "/Quickstart_Guide/index-foreman-deb.html"
```

Comment:
Looks like we need to reflect that Ubuntu 18.04 was dropped in Foreman 3.1.

It would be good to from from Puppet 6 to 7, but both are supported for now.

Reply:
> Looks like we need to reflect that Ubuntu 18.04 was dropped in Foreman 3.1.
https://github.com/theforeman/foreman-documentation/pull/969 does that

Reply:
I think this guide we should definitely fix as it will be heavily used. Once we convert the original manual, the first change would be to remove quickstart section from there and link these guides (as javascript was originally used). If you can help with this one this would be great.

Reply:
Merged & cherry-picking to 3.1.